tools/shell/embedded_shell.cpp

```diff
@@ -226,6 +226,46 @@ void highlight(char* buffer, char* resultBuf, uint32_t renderWidth, uint32_t cur
     strncpy(resultBuf, highlightBuffer.str().c_str(), LINENOISE_MAX_LINE - 1);
 }
 
+int damerauLevenshteinDistance(const std::string& s1, const std::string& s2) {
+    const size_t m = s1.size(), n = s2.size();
```

Comment:
You don't have to declare them as const, and also we use uint64_t instead of const size_t